README.md

```diff
@@ -283,6 +283,8 @@ For more information about the governance of the Node.js project, see
 **Isaac Z. Schlueter** &lt;i@izs.me&gt;
 * [joshgav](https://github.com/joshgav) -
 **Josh Gavant** &lt;josh.gavant@outlook.com&gt;
+* [trevnorris](https://github.com/trevnorris) -
+**Trevor Norris** &lt;trev.norris@gmail.com&gt;
```

Comment:
Hmmm...also added in the wrong place. (Lists are all alphabetical by GitHub handle.)  I'll fix both of these things in a moment, assuming I have push access on the branch...